src/conn/conn_api.c

```diff
@@ -2207,6 +2207,7 @@ __wt_timing_stress_config(WT_SESSION_IMPL *session, const char *cfg[])
       {"backup_rename", WT_TIMING_STRESS_BACKUP_RENAME},
       {"checkpoint_reserved_txnid_delay", WT_TIMING_STRESS_CHECKPOINT_RESERVED_TXNID_DELAY},
       {"checkpoint_slow", WT_TIMING_STRESS_CHECKPOINT_SLOW},
+      {"compact_slow", WT_TIMING_STRESS_COMPACT_SLOW},
```

Comment:
Can you use the flag in one of the csuite tests to verify the flag is working as expected.

Reply:
Yes, I will be using this flag in the `test_wt8246_compact_rts_data_correctness` test as part of the WT-8439 ticket.